applications/StructuralMechanicsApplication/custom_elements/small_displacement_bbar.cpp

```diff
@@ -112,7 +113,7 @@ void SmallDisplacementBbar::CalculateAll(
         if ( rRightHandSideVector.size() != mat_size )
             rRightHandSideVector.resize( mat_size, false );
 
-        rRightHandSideVector = ZeroVector( mat_size ); //resetting RHS
+        noalias(rRightHandSideVector) = ZeroVector( mat_size ); //resetting RHS
```

Comment:
just to tell (not really a review here, just to let you know) that you can use  the .clear()) method to do the same shorter